app/helpers/Intersect.js

```diff
@@ -127,7 +146,10 @@ export function intersectSetIntoBins(
     }
   }
 
-  return dayBins;
+  // Updates 'crossedPaths' field within location data if a user has crossed paths with infected case
+  if (newIntersections > 0) SetStoreData(LOCATION_DATA, localArray);
```

Comment:
Why this side effect here? It seems like we'd write localArray at the very end end when all iterations are done right?

Could this have a problem where it overwrites previous valid crossedPaths with empty crossedPaths because we're only halfway through the process?